### PR TITLE
Fix style of selected customer in product page

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | When you create a specific price for a customer, the display of the customer line doesn't have the good garbage.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-943
| How to test?  | Add a specific price to  a product and select customer.

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/pages/_product_page.scss [hunk 1]
```diff
@@ -1050,3 +1050,19 @@
     }
   }
 }
+
+#form_step2_specific_price_sp_id_customer-data {
+  border: rem(1px) solid #e5e5e5;
```
Comment (maximebiloe) [inline admin-dev/themes/new-theme/scss/pages/_product_page.scss]: We don't use use the `rem()` mixin anymore, it's deprecated by Bourbon.
Please, directly put the rem values.

File: src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/typeahead.html.twig [hunk 2]
```diff
@@ -274,6 +275,7 @@
                 html += sprintf('{% spaceless %}{{ template_collection|raw }}{% endspaceless %}', suggestion.{{ mapping_name }});
                 html += '<input type="hidden" name="{{ form.vars.full_name }}[data][]" value="' + value + '" />';
                 html += '</li>';
+                $('#{{ form.vars.id }}-data').show()
```
Comment (maximebiloe) [inline src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/typeahead.html.twig]: A little `;`, please?

**Review by maximebiloe — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/pages/_product_page.scss [hunk 3]
```diff
@@ -1050,3 +1050,19 @@
     }
   }
 }
+
+#form_step2_specific_price_sp_id_customer-data {
```
Comment (Quetzacoalt91) [inline admin-dev/themes/new-theme/scss/pages/_product_page.scss]: Why do we have specific style rules for this field?

If we use the UI kit as requested in the Forge issue, we should not have this kind of stuff in this PR.

File: admin-dev/themes/new-theme/public/theme.css [hunk 4]
```diff
@@ -10409,6 +10410,19 @@ ul.category-tree {
   #bulk-combinations-container .custom-width {
     width: 33%; } }
 
+#form_step2_specific_price_sp_id_customer-data {
+  border: 0.0625rem solid #e5e5e5;
+  border-radius: 0.125rem;
```
Comment (Quetzacoalt91) [inline admin-dev/themes/new-theme/public/theme.css]: Coming from the changes in theme.scss. See comment.

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Comment by maximebiloe** [comment]
Thank you @fatmaBouchekoua

**Pull request merged**